### DISCUSSION TITLE
Unique combinations in migrations docs

### DISCUSSION THREAD

**Discussion by igmltd** [laravel/framework#51569] [Q&A] [answered]
Hi, is there a reason why it is not noted in the docs that you can do unique groups in migrations?

 $table->unique(['table_id', 'booking_date', 'booking_time'], 'unique_booking');
 
 Is it not in the docs cos theres some other way do this? Or not recommended?
 
I couldn't find much on it, only codepilot's recommendation -  any info will be much appreciated,  thanks.

**Comment by rojtjo** [comment, marked as answer]
I think you might've just missed it in the docs. It states you can pass an array of columns to any of the index methods.

> You may even pass an array of columns to *an index method* to create a compound (or composite) index:

See: https://laravel.com/docs/11.x/migrations#creating-indexes

**Reply by igmltd** [reply]
You're right, I did miss it - I was looking for what I wanted to see rather than read between the lines.
You've given me some light on my approach to the docs in general - got more than I bargained for!

Thanks ; )
